abs.yaml

```diff
@@ -0,0 +1,5 @@
+aggregate_branch: 3.12
+
+channels:
```

Comment:
These two are now deployed as of last Friday.  So we shouldn't need the channels here anymore.

Reply:
removed